internal/usecase/calculate_diff.go

```diff
@@ -0,0 +1,45 @@
+package usecase
+
+import (
+	"github.com/shopspring/decimal"
+	"github.com/supressionstop/xenking_test_1/internal/usecase/dto"
+)
+
+type CalculateDiffUseCase struct{}
```

Comment:
If we are already in `usecase` package we don't need to append the suffix `UseCase` at the type

Reply:
TODO: remove package suffixes.